torch/autograd/variable.py

```diff
@@ -363,6 +363,9 @@ def norm(self, p=2, dim=None, keepdim=False):
         else:
             return super(Variable, self).norm(p, dim, keepdim)
 
+    def det(self):
```

Comment:
Shouldn't this be defined as a native function as well?

Reply:
Defining this as a native function sounds like a much better idea!